modules/icgc-argo-workflows/prep/sample/resources/usr/bin/main.py

```diff
@@ -337,8 +345,8 @@ def main():
           sys.exit("Error: not supported input file format")
       with open(output_sample_sheet, 'w', newline='') as f:
         csvwriter = csv.writer(f, delimiter=',')
-        csvwriter.writerow(['analysis_type','study_id','patient','sex','status','sample','cram','crai', 'analysis_json'])
-        csvwriter.writerow([analysis_type, study_id, donor_id, sex, status, sample_id, cram, crai, metadata_json])
+        csvwriter.writerow(['analysis_type','study_id','patient','sex','status','sample','cram','crai',"genome_build",'experiment'])
+        csvwriter.writerow([analysis_type, study_id, donor_id, sex, status, sample_id, cram, crai, experiment,genome_build, metadata_json])
```

Comment:
- `experiment` and `genome_build` should switch position?
- `metadata_json` is missing field definition